app/views/samples/index.haml

```diff
@@ -45,6 +48,7 @@
                 %td= sample.inactivation_method
                 %td= sample.specimen_role_description
                 %td= sample.date_produced_description
+                %td{id: 'sample_is_qc', class: 'nodisplay'}= sample.is_quality_control?
```

Comment:
Does this information need to be expressed in a hidden cell? How about using a simple `data` attribute on the row instead?